src/platform/unittest/TlsTest.cpp

```diff
@@ -1930,8 +1926,7 @@ TEST_F(TlsTest, ClientCertificateDeferValidation)
     DoHandshake(ServerContext, ClientContext);
 }
 
-#ifndef _WIN32
-
+#ifdef QUIC_ENABLE_CA_CERTIFICATE_FILE
```

Comment:
But you still need to define this, right?

Reply:
And the define should specifically specify `_TESTS`

Reply:
Yes, good points.